airflow/configuration.py

```diff
@@ -525,14 +532,11 @@ def _get_env_var_option(self, section, key):
     def _get_cmd_option(self, section, key):
         fallback_key = key + '_cmd'
         # if this is a valid command key...
-        if (section, key) in AirflowConfigParser.as_command_stdout:
-            # if the original key is present, return it no matter what
-            if self.has_option(section, key):
-                return ConfigParser.get(self, section, key)
-            # otherwise, execute the fallback key
-            elif self.has_option(section, fallback_key):
-                command = self.get(section, fallback_key)
-                return run_command(command)
+        if (
+                (section, key) in self.as_command_stdout
+                and self.has_option(section, fallback_key)):
+            command = self.get(section, fallback_key)
```

Comment:
This method is being called inside of `self.get`, running all the way through again puts us at risk of introducing infinite loops later; let's use `super` here:

```python
command = super(AirflowConfigParser, self).get(section, fallback_key)
```